openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -226,7 +234,9 @@ const DatabaseSchemaPage: FunctionComponent = () => {
       const { description: schemaDescription = '' } = response;
       setDatabaseSchema(response);
       setDescription(schemaDescription);
-      setShowDeletedTables(response.deleted ?? false);
+      if (tableFilters.showDeletedTables === undefined) {
+        setFilter('showDeletedTables', response.deleted ?? false);
+      }
```

Comment:
We don't want to set it false we can leverage on default value in absence of filter